modules/spring-config/src/main/java/org/mule/runtime/config/spring/factories/TryProcessorFactoryBean.java

```diff
@@ -55,32 +51,12 @@ public void setMessageProcessors(List messageProcessors) {
 
   @Override
   public Object getObject() throws Exception {
-    DefaultMessageProcessorChainBuilder builder = new DefaultMessageProcessorChainBuilder();
-    builder.setName("'transaction' child processor chain");
-    TransactionalInterceptingMessageProcessor txProcessor = new TransactionalInterceptingMessageProcessor();
+    TryMessageProcessor txProcessor = new TryMessageProcessor();
     txProcessor.setAnnotations(getAnnotations());
     txProcessor.setExceptionListener(this.exceptionListener);
     txProcessor.setTransactionConfig(createTransactionConfig(this.transactionalAction, this.transactionType));
-    builder.chain(txProcessor);
```

Comment:
Some isn't required, the rest is moved to TryMessageProcessor which becomes a scope instead of an interceptor.